src/Microsoft.VisualStudio.Web.CodeGeneration.EntityFrameworkCore/Templates/DbContext/NewDbContextFactory.cshtml

```diff
@@ -21,6 +21,6 @@ namespace @Model.DbContextNamespace
     public class @factoryClassName : IDbContextFactory<@Model.DbContextTypeName>
     {
         public @Model.DbContextTypeName Create(string[] args) =>
-            @(Model.ProgramTypeName).BuildWebHost(args).Services.GetRequiredService<@Model.DbContextTypeName>();
+            @(Model.ProgramTypeName).BuildWebHost(args).Services.CreateScope().ServiceProvider.GetRequiredService<@Model.DbContextTypeName>();
```

Comment:
Created https://github.com/aspnet/Scaffolding/issues/485. 
Will need to keep this for now so the tests don't break.